docs/Onboarding.md

```diff
@@ -33,6 +33,20 @@ Paste the following line into the `.env` file and replace `<USERNAME>` with your
 To test that things are working, run `npm run start` in the CLI and go to an example API route.
 Example: `localhost:3000/.netlify/functions/users`.
 
+If the application does not launch, you can replace the line in the `.env` file with the following.
+`DATABASE_URL="postgresql://postgres:<PASSWORD>@localhost:5432/nerpm?schema=public"`
+Change `<PASSWORD>` to your `postgres` database password.
+
+Test again to ensure that the application launches correctly.
+
+### Initial Database Migration
+
+In order to run the database for the first time, you will need to execute the following commands in the CLI.
+
+Run `npm prisma:migrate`, followed by `npm prisma:seed`, and then `npm prisma:generate`.
```

Comment:
is prisma generate really required? also if you think about it in the context of the steps, they were instructed to run the application on line 33. with the local development database, the example and test they are instructed to do on line 33 and 34 will not work until after they do the initial database migration step. perhaps the instructions should be shifted a bit so that we have these three subheadings instead of the project preparation subheading 
- env setup
- initial database migration
- run and test that it works